app/assets/stylesheets/components/page_specific/_link_list.scss

```diff
@@ -0,0 +1,42 @@
+/*
+  Link list component
+  Used on category pages to list out yellow underlined links
+*/
+.link-list {
+  @include body(22, 40);
+  font-weight: bold;
+
+  .l-category--image & {
+    @include respond-to($mq-m) {
+      width: 60%;
+    }
+  }
+}
+
+.link-list__item {
+  display: inline;
+}
+
+.link-list__item:before {
+  content: none !important;
+}
+
+.link-list__item:after {
+  content: ", ";
+}
+ 
+.link-list__item:last-child:after {
+  content: "";
```

Comment:
You've got `content: none;` above and `content: ""` here. Do these do the same thing? If so could prob choose just one of them.